app/controllers/lists_controller.rb

```diff
@@ -1,68 +1,35 @@
 class ListsController < ApplicationController
   before_action :authenticate_user!
-  before_action :set_list, only: [:show, :edit, :update, :destroy]
-
-  def index
-    @lists = policy_scope(List).all
-  end
+  before_action :set_list, only: [:show, :destroy]
 
   def show
     authorize @list
     @list_ingredients = @list.list_ingredients
     @list_ingredient = ListIngredient.new
-
-  end
-
-  def new
-    @list = List.new
-    authorize @list
-  end
-
-  def create
-    @list = List.new(list_params)
-    @list.user = current_user
-
-    authorize @list
-
-    if @list.save
-      redirect_to lists_path
-    else
-      render :new, status: :unprocessable_entity
-    end
-  end
-
-  def destroy
-    authorize @list
```

Comment:
why is the destroy method deleted, if the before_action is referring to it?

Reply:
it is not :) and we have a permanent List now so cannot delete it

Reply:
i realized that later on in the code :) sweet